src/renderer/pages/RecordingPage.vue

```diff
@@ -254,29 +269,39 @@ export default defineComponent({
         })
       })
 
+    // Needs to wait until the metronomes are saved because they are created on the page switch
+    await new Promise(r => setTimeout(r, 500));
+    
     // Add checkboxes for each audio and metronome track to be played back
     const playbackTracks = <HTMLDivElement> this.$refs.playbackTracks;
     ipcRenderer.invoke("get-option", "audioTracks").then((recordings: string) => {
-      ipcRenderer.invoke("get-option", "clickTracks").then((metronomes: string) => {
-        const audiofiles = (<string[]> JSON.parse(recordings)).concat(JSON.parse(metronomes));
-        audiofiles.forEach(function(file:string) {
-          var checkbox = document.createElement('input');
-          checkbox.className = 'tickbox';
-          checkbox.type = 'checkbox';
-          checkbox.value = file;
-
-          var header = document.createElement('h3');
-          header.textContent = file.substr(0, file.indexOf(".w"));
-
-          playbackTracks.appendChild(checkbox);
-          playbackTracks.appendChild(header);
+      const audiofiles = <string[]> JSON.parse(recordings);
+      ipcRenderer.invoke("get-temp-directory").then(dir => {
+
+        fs.readdir(dir, (err, files) => {
+          files.forEach(file => {
+            audiofiles.push(file);
+          });
+
+          audiofiles.forEach(function(file:string) {
+            var checkbox = document.createElement('input');
+            checkbox.className = 'tickbox';
+            checkbox.type = 'checkbox';
+            checkbox.value = file;
+
+            var header = document.createElement('h3');
+            header.textContent = file.substr(0, file.indexOf(".w"));
+
+            playbackTracks.appendChild(checkbox);
+            playbackTracks.appendChild(header);
+          });
         });
       });
     });
 
     video.autoplay = true;
     this.startStreams();
-  }
+  },
```

Comment:
I love the overall code quality, it matches the repository greatly, could you run `npm run lint:fix` to make sure these are removed?

Reply:
hahaha